django_lti_tool_provider/migrations/0002_reduce_custom_key_length.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.db.models import Max
+from django.db.models.functions import Length
+
+
+def check_field_max_length_lteq_190(apps, schema_editor):
+    """
+    Check to see if we've got any data in the table that prevents us from
+    migrating the length of the column down to 190; if so, raise an exception.
+    """
+    LtiUserData = apps.get_model('django_lti_tool_provider', 'LtiUserData')
+    max_custom_key_length = LtiUserData.objects.aggregate(length=Max(Length('custom_key')))['length']
+    print(max_custom_key_length)
```

Comment:
Should remove this `print`. Should also put the "max length found" and "expected max length" in the ValueError string below.